### ISSUE TITLE
Error Adding my own XFTP server

### ISSUE THREAD

**Issue by rmsalinas** [simplex-chat/simplexmq#811]
I have succesfully configured my own SMP and XFTP server in Ubuntu. In the android app, I am able to add the SMP server. However, when I try to add the XFTP server I have the following problem.

I am able to add it, and test it. But at the moment of saving them I get the message (translate from spanish) 

Error saving SMP servers: Make sure the addresses of the XFTP server have the correct format, and separated by commas and are not duplicated. 

I believe this is a bug in the program at the moment of saving them. 
Thanks for your excellent work!

**Comment by egreek** [comment]
I faced with the same problem and found that SimpleX doesn't save your XFTP server config if it's IP address equals to IP address in the SMP config. I don't really know is it a bug or feature, but you can try to solve it by using additional IP address.

**Comment by spaced4ndy** [comment]
If servers are hosted on the same IP address, try explicitly specifying ports in server addresses you're configuring in your client. It's `<smp/xftp>://<fingerprint>@<hostname>:<port>` - so you'd add `:<port>` to the end of the addresses. It should be 5223 for smp and 443 for xftp by default, but you can double check in /etc/opt/simplex/smp-server.ini and /etc/opt/simplex-xftp/file-server.ini.

**Comment by egreek** [comment]
> If servers are hosted on the same IP address, try explicitly specifying ports in server addresses you're configuring in your client. It's `<smp/xftp>://<fingerprint>@<hostname>:<port>` - so you'd add `:<port>` to the end of the addresses. It should be 5223 for smp and 443 for xftp by default, but you can double check in /etc/opt/simplex/smp-server.ini and /etc/opt/simplex-xftp/file-server.ini.

I also found it's working yesterday. This solution is correct.

**Comment by epoberezkin** [comment]
yes, we still didn't address the restriction that these two servers cannot be on the same host without explicitly specifying the port.

**Comment by nicfab** [comment]
I have the same issue, although I set data correctly in `/etc/opt/simplex-xftp/file-server.ini`.

**Comment by davilico05** [comment]
> try explicitly specifying ports in server addresses you're configuring in your client. It's `<smp/xftp>://<fingerprint>@<hostname>:<port>` - so you'd add `:<port>` to the end of the addresses.

Same issue with the app solved adding the  `:<port>`
